lambdas/data-migration2/tests/test-executions.js

```diff
@@ -474,10 +485,11 @@ test.serial('migrateExecutions processes all non-failing records', async (t) =>
   await Promise.all([
     // Have to use Dynamo client directly because creating
     // via model won't allow creation of an invalid record
-    dynamodbDocClient().put({
-      TableName: process.env.ExecutionsTable,
-      Item: newExecution,
-    }).promise(),
+    // dynamodbDocClient().put({
```

Comment:
This looks like it should be removed.

Reply:
dfda99d54ee1397ee7ab8be22337ffa0a85dd651